package.json

```diff
@@ -6,6 +6,7 @@
   "types": "index.d.ts",
   "scripts": {
     "docs": "docsify serve",
+    "tap": "tap",
```

Comment:
why this change?

Reply:
Ah sorry, I'll undo this (added to run a single test e.g. `npm run tap test/subscription-connection.js`)